api/types/types.go

```diff
@@ -168,6 +168,7 @@ const (
 	TokenGeneratedBy            string = BaseQualifier + "/generated-by"
 	SiteVersion                 string = BaseQualifier + "/site-version"
 	TokenCost                   string = BaseQualifier + "/cost"
+	TokenDcc                    string = BaseQualifier + "/dcc"
```

Comment:
As above, particularly for the actual annotation key, which is user visible, lets have something a bit less cryptic.